### PR TITLE
Adds list UI to be reused

### PR DESCRIPTION
**Pull request by belcherj**
This PR creates a reusable List UI that can be used within the WooCommerce extension.

You can test this by going here:  http://calypso.localhost:3000/store/settings/{slug}/payments

### CODE REVIEW

**Comment by kellychoffman** [comment]
> Formatting is not complete on this PR. I would like feedback on the underlying architecture.

Didn't see this at first. Would you rather I hold off styling?

**Comment by belcherj** [comment]
@kellychoffman this commit relates to the list under the offline section. Feel free to style! We can remove fade in another PR

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ListTd from '../../../components/list/list-td';
+
+class PaymentMethodRow extends Component {
```
Comment (DanReyLop) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Where do we stand on using `class` components vs `function(){}` components?

In `woocommerce-services` we have agreed to always use function components when possible (i.e. when the component doesn't have `state`), but I see that `wp-calypso` doesn't have a convention in that regard.

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js [hunk 2]
```diff
@@ -25,7 +52,17 @@ class SettingsPaymentsOffSite extends Component {
...
 					} />
-				<Card></Card>
+					<ListTable>
+						<ListHeader>
+							<ListTd width="30">Method</ListTd>
+							<ListTd width="45">Fees</ListTd>
+							<ListTd width="25"></ListTd>
+						</ListHeader>
+						<ListRows>
+							{ methods && methods.map( ( m ) => this.renderMethodRow( m ) ) }
```
Comment (DanReyLop) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: You can shorten this to `methods && methods.map( this.renderMethodRow )`

File: client/extensions/woocommerce/components/list/style.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import 'list-header/style';
```
Comment (DanReyLop) [inline client/extensions/woocommerce/components/list/style.scss]: All these CSS files are so small that I would rather have everything be in the same `list/style.scss` file. If it fits in an editor screen, then one file is better than 5 IMHO :)

File: client/extensions/woocommerce/components/list/list-table/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ */
+import Card from 'components/card';
+
+class ListTable extends Component {
+
+	render() {
+		const { children } = this.props;
+		return (
+			<Card className="list-table">
+				<table className="list-table__table">
```
Comment (DanReyLop) [inline client/extensions/woocommerce/components/list/list-table/index.js]: Using `<table>` gives me the chills. I know this is tabular data, but still... Let's see what the designers say of this, I don't know if `<table>`s are still considered super-evil.

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js [hunk 5]
```diff
@@ -25,7 +52,17 @@ class SettingsPaymentsOffSite extends Component {
 							'information'
 						)
 					} />
-				<Card></Card>
+					<ListTable>
+						<ListHeader>
+							<ListTd width="30">Method</ListTd>
```
Comment (DanReyLop) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: You named the components `ListTd`, `ListTable` etc. I think they should have more semantic names, instead of blatantly giving away that they represent an HTML `<table>` underneath. What about `ListRowField` instead of `ListTd`, for example?

Comment (DanReyLop) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: Also, why not just putting the full `"30%"` here? It makes it clearer at a glance what those "widths" mean.

File: client/extensions/woocommerce/components/list/style.scss (hunk 3 above)
Comment (belcherj) [inline client/extensions/woocommerce/components/list/style.scss]: I suspect that there will be more css once it is styled.  I personally like it separated and if css modules or inlining css becomes a thing in calypso having it separate makes that easier.

File: client/extensions/woocommerce/components/list/list-table/index.js (hunk 4 above)
Comment (belcherj) [inline client/extensions/woocommerce/components/list/list-table/index.js]: Tables are not considered evil as long as they are used semantically. In this case table is correct

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js (hunk 5 above)
Comment (belcherj) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: I agree with both statements

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js (hunk 1 above)
Comment (belcherj) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: They should all be switched to use arrow functions.  Thank you.

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+const PaymentMethodRow = ( { method, translate } ) => {
+	return (
+		<tr className="payments__method-row">
+			<ListTd>
+				{
+					method.suggested &&
+					(
+						<p className="payments__method-suggested">
+							{ translate( 'Suggested Method' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jameskoster** [comment]
For the settings row, could we add the following class to the last column (the one which contains the setup button); `.payments__setup-column`

Also, could we add the `.is-compact` class to the button in that column?

**Comment by kellychoffman** [comment]
> Also, could we add the .is-compact class to the button in that column?

The `Set up` button? I'm wondering if that should stay big because its the primary action of the section.

**Comment by kellychoffman** [comment]
@belcherj: will the component be used in /shipping? (cc: @marcinbot) I feel like we could benefit from having the same exact styles in each.

For example, they are both different and I'd have to style them both uniquely. It would be nice if we could do this in one place.

screen shot 2017-05-12 at 11 48 47 am
screen shot 2017-05-12 at 12 05 15 pm

**Comment by kellychoffman** [comment]
> Also, could we add the .is-compact class to the button in that column?

done.

**Comment by jameskoster** [comment]
> done.

TIL!

> For the settings row, could we add the following class to the last column (the one which contains the setup button); .payments__setup-column

Just that ^^ outstanding then @belcherj. Although I agree with Kelly and Marcin re using the same classes / styles for the shipping list. So maybe use something like `.list__action-column` rather than `.payments__setup-column`. I suppose `.list__*` might be too generic. `.settings-list__*`? I'll leave that to you :)

**Comment by belcherj** [comment]
@jameskoster I think that adding those classes is out of scope for this PR.  I think the PR that switches out shipping to use this would be better suited.

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+				<p>
+					<a href={ method.information }>
+						{ translate( 'More Information' ) }
+					</a>
+				</p>
+			</ListRowField>
+			<ListRowField>
+				<Button compact>
+					{
+						translate( 'Set Up' )
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Set up' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+				<p>
+					<a href={ method.information }>
+						{ translate( 'More Information' ) }
+					</a>
+				</p>
+			</ListRowField>
+			<ListRowField className="payments__setup-column">
+				<Button compact>
+					{
+						translate( 'Set Up' )
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Set up' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jameskoster** [comment]
Styling all looks good to me 👍

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+					}
+				</Button>
+			</ListRowField>
+		</tr>
+	);
+};
+
+PaymentMethodRow.propTypes = {
+	method: PropTypes.shape( {
+		name: PropTypes.string.isRequired,
```
Comment (allendav) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Minor - please consider using `label` instead of `name` for consistency with other parts of the code

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+		suggested: PropTypes.bool.isRequired,
```
Comment (allendav) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Please consider `isSuggestedMethod` or `isSuggested` to make code just a little bit more readable/understandable

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+		fee: PropTypes.string.isRequired,
+		information: PropTypes.string.isRequired,
```
Comment (allendav) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Please consider `informationURL` or something similar - at first glance I thought this was a description attribute.

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js [hunk 12]
```diff
@@ -7,13 +7,40 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import Card from 'components/card';
+
 import ExtendedHeader from '../../../components/extended-header';
+import ListHeader from '../../../components/list/list-header';
+import ListRows from '../../../components/list/list-rows';
+import ListTable from '../../../components/list/list-table';
+import ListRowField from '../../../components/list/list-row-field';
```
Comment (allendav) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: Please alphabetize imports

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js [hunk 13]
```diff
@@ -25,7 +52,17 @@ class SettingsPaymentsOffSite extends Component {
 							'information'
 						)
 					} />
-				<Card></Card>
+					<ListTable>
+						<ListHeader>
+							<ListRowField width="30%">Method</ListRowField>
```
Comment (allendav) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: Could we pass in a className prop and thereby use CSS instead please? That way we can keep styling all in one place.

File: client/extensions/woocommerce/components/list/list-row-field/index.js [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+
+const ListRowField = ( { children, width } ) => {
+	return (
+		<td className="list-row-field" style={ { width: width } }>
+			{ children }
+		</td>
+	);
+};
+
+export default ListRowField;
```
Comment (allendav) [inline client/extensions/woocommerce/components/list/list-row-field/index.js]: Please consider enforcing proptypes

File: client/extensions/woocommerce/components/list/list-rows/index.js [hunk 15]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (allendav) [inline client/extensions/woocommerce/components/list/list-rows/index.js]: Does this need to be a separate component? Seems kinda overkill

File: client/extensions/woocommerce/components/list/list-row/index.js [hunk 16]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (allendav) [inline client/extensions/woocommerce/components/list/list-row/index.js]: This component isn't actually used anywhere, is it?

File: client/extensions/woocommerce/components/list/list-header/index.js [hunk 17]
```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+const ListHeader = ( { children } ) => {
```
Comment (allendav) [inline client/extensions/woocommerce/components/list/list-header/index.js]: Please consider naming this component `ListTableHeader` instead to make it clearer that it belongs in the context of a table (because it emits a `thead`)

**Comment by allendav** [comment]
Having the button label all uppercase doesn't match buttons elsewhere in our UI. Is that really what we want?

cc @jameskoster @kellychoffman 

screen shot 2017-05-16 at 8 59 50 am

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js (hunk 9 above)
Comment (justinshreve) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: If this comes from `https://woo.dev/wp-json/wc/v2/payment_gateways` then `title` might actually make more sense here (to match the data coming through). However, the rest of this object looks like stuff we need hardcoded, so I guess we have a little flexibility on what things are named.

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js [hunk 18]
```diff
@@ -7,13 +7,40 @@ import { localize } from 'i18n-calypso';
...
+import ListTable from '../../../components/list/list-table';
+import ListRowField from '../../../components/list/list-row-field';
+import PaymentMethodRow from './payment-method-row';
 
 class SettingsPaymentsOffSite extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state and real data
+		this.state = {
```
Comment (justinshreve) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: Minor since this is temporary, but since props are not needed to set the state, the constructor could be dropped.

[Example](https://github.com/Automattic/wp-calypso/commit/f6982493ca56823d49609e1be9ee32d1acae78c7#diff-de58a3db9f3b73767772821d720b47a6R22).

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js (hunk 13 above)
Comment (justinshreve) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: These should also be wrapped in `translate`.

File: client/extensions/woocommerce/components/list/list-rows/index.js (hunk 15 above)
Comment (belcherj) [inline client/extensions/woocommerce/components/list/list-rows/index.js]: This allows us to fully abstract out the table html and also style the th within this component.  If we do not have this component then we would need to style the tbody somewhere else.

File: client/extensions/woocommerce/components/list/list-row/index.js (hunk 16 above)
Comment (belcherj) [inline client/extensions/woocommerce/components/list/list-row/index.js]: This is not used but would be used instead of payment-method-row.  I can delete but I felt it was needed to complete the set of table components

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js (hunk 9 above)
Comment (belcherj) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: These are just placeholders and I suspect they will change with addition of the api.

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js (hunk 12 above)
Comment (belcherj) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: Love me some alphabetizing, done

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+				<p>
+					<a href={ method.information }>
+						{ translate( 'More Information' ) }
+					</a>
+				</p>
+			</TableRowField>
+			<TableRowField className="payments__setup-column">
+				<Button compact>
+					{
+						translate( 'Set Up' )
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Set up' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js [hunk 20]
```diff
@@ -0,0 +1,55 @@
...
+
+const PaymentMethodRow = ( { method, translate } ) => {
+	return (
+		<tr className="payments__method-row">
+			<TableRowField>
+				{
+					method.isSuggested &&
+					(
+						<p className="payments__method-suggested">
+							{ translate( 'Suggested Method' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by kellychoffman** [comment]
> Having the button label all uppercase doesn't match buttons elsewhere in our UI. Is that really what we want?

This is the calypso pattern for compact buttons. We have them in a quite a few places now in both the shipping and product pages.

**Comment by kellychoffman** [comment]
Question: Is table layout necessary or is there a reason why we can't use a div based layout like the one in /shipping?

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js (hunk 20 above)
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by kellychoffman** [comment]
> Styling all looks good to me

I fixed the header so that it matches the work we did on shipping. @jameskoster - I didn't get to style the Method section. The type feels really small overall and could use some hierarchy, similar to what /shipping currently has.

**Comment by jameskoster** [comment]
> I fixed the header so that it matches the work we did on shipping. @jameskoster - I didn't get to style the Method section. The type feels really small overall and could use some hierarchy, similar to what /shipping currently has.

Looking at Shipping again, you're right. I've made some tweaks. Should be good now.

@belcherj one issue; the classes we've added to various components aren't actually being applied in the markup. Examples;

* https://github.com/Automattic/wp-calypso/blob/7ad2cd7752fdf5d3f9a85b5a9cc2867efa34eb46/client/extensions/woocommerce/app/settings/payments/payment-method-row.js#L35
* https://github.com/Automattic/wp-calypso/blob/7ad2cd7752fdf5d3f9a85b5a9cc2867efa34eb46/client/extensions/woocommerce/app/settings/payments/payment-method-row.js#L27
* https://github.com/Automattic/wp-calypso/blob/7ad2cd7752fdf5d3f9a85b5a9cc2867efa34eb46/client/extensions/woocommerce/app/settings/payments/payment-method-row.js#L15

The styling will be a _little_ off until this is addressed.

File: client/extensions/woocommerce/app/settings/payments/payment-method-row.js (hunk 20 above)
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-row.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by kellychoffman** [comment]
> Question: Is table layout necessary or is there a reason why we can't use a div based layout like the one in /shipping?

I followed up in Slack about this, but using a table will make the mobile/small screen experience difficult to use.

**Comment by coderkevin** [comment]
@kellychoffman and @jameskoster, when I implemented something similar in `wc-synchrotron` here: https://github.com/Automattic/wc-synchrotron/tree/master/client/app/product-list

I believe it was a design decision to use `<ul>` and `<li>` instead of `<table>`. Does that reasoning then also apply here? Or is this different?

**Comment by jameskoster** [comment]
Like Kelly said, using markup other than `table` will make things easier on small screens going forwards.

Having said that, the table is not a disaster currently;

table

That's not to say it wouldn't be a disaster in other applications of this component though.

For now a table is fine, but in an ideal world we'd use a list I think.

**Comment by jameskoster** [comment]
The fade at the end of the section header should be removed;

fade

Removing the `.section-header__label` class in the screenshot below would fix;

remove-class

But it seems that's coming from another component? I'll leave that one to your @belcherj xD

**Comment by kellychoffman** [comment]
Re: table discussion. Would this take extra time? If we don't do this now, I'm afraid it won't ever get done and since its a component, we'll have a lot of extra styling work to do down the road, that will effect a lot of areas. I'd rather us not do a double work and fix it before we start applying this to a bunch of places. We'll also have a harder time styling now with a table. Jay - you're right that it doesn't look terrible, but we could make it a lot better.

**Comment by jameskoster** [comment]
Oh, to be clear, I'd prefer we didn't use a table.

> Would this take extra time?

It all comes down to this really.

File: client/extensions/woocommerce/components/list/list-row/index.js (hunk 16 above)
Comment (allendav) [inline client/extensions/woocommerce/components/list/list-row/index.js]: My vote is to not check in unused code - maybe throw it in a public gist (and link to it from here) for safekeeping?

File: client/extensions/woocommerce/components/list/list-rows/index.js (hunk 15 above)
Comment (allendav) [inline client/extensions/woocommerce/components/list/list-rows/index.js]: But do we need to fully abstract out the markup at this time? Also, the style rationale is a consequence of wanting to not combine styles in a single scss for the overall list component, no?

My vote would be to not break things down this far, but if you prefer it this way, that's fine too :)

**Comment by belcherj** [comment]
This commit is ready for review.  Changed from Table to List (UI)

File: client/extensions/woocommerce/app/settings/payments/payment-method-item.js [hunk 21]
```diff
@@ -0,0 +1,56 @@
...
+				<p className="payments__method-information">
+					<a href={ method.information }>
+						{ translate( 'More Information' ) }
+					</a>
+				</p>
+			</ListItemField>
+			<ListItemField>
+				<Button compact>
+					{
+						translate( 'Set Up' )
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-item.js]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Set up' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/settings/payments/payment-method-item.js [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+
+const PaymentMethodItem = ( { method, translate } ) => {
+	return (
+		<ListItem>
+			<ListItemField>
+				{
+					method.isSuggested &&
+					(
+						<p className="payments__method-suggested">
+							{ translate( 'Suggested Method' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-item.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by belcherj** [comment]
@jameskoster The fade comes from here:  https://github.com/Automattic/wp-calypso/blob/583f7940592871d10d528233d916cc06f9251be7/client/components/section-header/style.scss#L22

I think we will need to modify that element and allow for passing of a flag for no-fade="true"

**Comment by kellychoffman** [comment]
👍   Looks good!

**Comment by kellychoffman** [comment]
Going to tweak the mobile view styles when there are more payment methods to show.

File: client/extensions/woocommerce/app/settings/payments/payment-method-item.js (hunk 21 above)
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-item.js]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Set up' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/settings/payments/payment-method-item.js (hunk 22 above)
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-item.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jameskoster** [comment]
> I think we will need to modify that element and allow for passing of a flag for no-fade="true"

I opened an [issue](https://github.com/Automattic/wp-calypso/issues/14454)

File: client/extensions/woocommerce/app/settings/payments/payment-method-item.js (hunk 21 above)
Comment (justinshreve) [inline client/extensions/woocommerce/app/settings/payments/payment-method-item.js]: Minor: Reduce this translate call down to 1 line. Also looks like a good suggestion by the i18n bot.

File: client/extensions/woocommerce/app/settings/payments/style.scss [hunk 23]
```diff
@@ -0,0 +1,37 @@
...
+
+	.list-item-field:nth-child( 2 ) {
+		flex: 0 0 45%;
+	}
+
+	.list-item-field:nth-child( 3 ) {
+		flex: 0 0 20%;
+
+		.button {
+				align-self: center;
```
Comment (justinshreve) [inline client/extensions/woocommerce/app/settings/payments/style.scss]: Minor: Looks like an extra indent here.

File: client/extensions/woocommerce/app/settings/payments/payments-off-site.js [hunk 24]
```diff
@@ -7,15 +7,37 @@ import { localize } from 'i18n-calypso';
...
+import PaymentMethodItem from './payment-method-item';
 
 class SettingsPaymentsOffSite extends Component {
 
+	state = {
+		methods: [
+			{
+				label: 'PayPal Standard',
+				isSuggested: true,
+				fee: '2.9% + 30c per transaction',
```
Comment (coderkevin) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: Is there no way to get this information from an API? What happens when PayPal changes their fees, we have to update this code then?

Comment (belcherj) [inline client/extensions/woocommerce/app/settings/payments/payments-off-site.js]: This is mock data, there is follow up PR.  Also that data does not come over the API.

File: client/extensions/woocommerce/app/settings/payments/payment-method-item.js [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-item.js]: Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).
